doc/man3/X509_VERIFY_PARAM_set_flags.pod

```diff
@@ -264,8 +264,9 @@ they are enabled.
 If B<X509_V_FLAG_USE_DELTAS> is set delta CRLs (if present) are used to
 determine certificate status. If not set deltas are ignored.
 
-B<X509_V_FLAG_CHECK_SS_SIGNATURE> enables checking of the root CA self signed
-certificate signature. By default this check is disabled because it doesn't
+B<X509_V_FLAG_CHECK_SS_SIGNATURE> requires verifying the signature of the last
+certificate in a chain, which typically is a trusted self-signed root CA
+certificate. By default this check is disabled because it does not
```

Comment:
See above, only self-signed certs get checked in this case. It is impossible to check the signature otherwise.

Reply:
I've changed this as well, analogously to above.